app/build.gradle

```diff
@@ -35,13 +35,14 @@ dependencies {
     implementation "androidx.navigation:navigation-ui:$nav_version"
     implementation 'androidx.legacy:legacy-support-v4:1.0.0'
     implementation 'com.google.android.material:material:1.2.0'
+    implementation 'de.hdodenhof:circleimageview:2.2.0'
```

Comment:
Hmm, this does not look like a standard library. Can you do the same thing with the standard toolkit?

Reply:
Yes, I did.